src/NotFound.svelte

```diff
@@ -1,11 +1,11 @@
 <script lang="ts">
+  import { pop } from "@app/router";
```

Comment:
Can I suggest to clarify to future devs here maybe to make it clearer use

```suggestion
  import router from "@app/router";
  
  <Button variant="foreground" on:click={router.back}>Back</Button>
```